tyaml/dumper.py

```diff
@@ -16,4 +18,42 @@ def represent_object(self, data):
         return self.represent_dict(result_dict)
 
 
-SpecialDumper.add_multi_representer(object, SpecialDumper.represent_object)
+OutputOptionalNullsDumper.add_multi_representer(object, OutputOptionalNullsDumper.represent_object)
```

Comment:
Maybe somehow shorten the class names, especially `NoOutputOptionalNullsDumper`, e.g. `DumperOmitNulls` and `DumperKeepNulls`.